src/constraint_aware_append.c

```diff
@@ -360,13 +373,80 @@ constraint_aware_append_plan_create(PlannerInfo *root, RelOptInfo *rel, struct C
 	CustomScan *cscan = makeNode(CustomScan);
 	Plan *subplan = linitial(custom_plans);
 	RangeTblEntry *rte = planner_rt_fetch(rel->relid, root);
+	List *chunk_ris = NIL;
```

Comment:
I think this name is a bit confusing since these are not restrict info but just the clauses. Maybe `chunk_ri_clauses`?

Reply:
changed